browser/mocha/replace.js

```diff
@@ -22,6 +22,10 @@ extendInterfaces('replace', function(context, teardown) {
         // implementation for later:
         var originalInstanceTemplate = Polymer.Base.instanceTemplate;
 
+        if (Polymer.Base.instanceTemplate.isSinonProxy) {
```

Comment:
I added a comment in the last PR before closing it:

Further investigation shows that Sinon's API does not seem to support replacing the wrapped spy. Also looking at their GH for this function, it seems as if replacing it manually would just repeat their already-written code. I recommend that we restore if it is already wrapped in a spy. (stay the same)

WDYT?